src/Identity/UI/src/Areas/Identity/Pages/V4/Account/Manage/Index.cshtml.cs

```diff
@@ -103,7 +103,7 @@ public override async Task<IActionResult> OnPostAsync()
         {
             if (!ModelState.IsValid)
             {
-                return Page();
+                return await OnGetAsync();
```

Comment:
This is weird, does `Page()` simply doesn't work? /cc: @rynowak Is there any downside to doing this instead of Page() which is AFAIK the recommended pattern for this?

Reply:
The issue is that Page() loses all the current state, so rather than duplicating the logic in Get here to populate the model, it seems easier to just return the Get repsonse... 🤷‍♂

Reply:
The refresh issue is from here https://github.com/aspnet/AspNetCore/issues/11749

Reply:
ping @javiercn @rynowak I'm happy to fix this another way if this is too weird